### PR TITLE
Provide more information when plugin cannot be installed in bundle

### PR DESCRIPTION
**Pull request by Czaki**
# Description

This PR adds information that plugin cannot be installed because is not available on conda-forge.  

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

From zulip https://napari.zulipchat.com/#narrow/stream/212875-general/topic/Plugin.20install.20in.20the.20bundled.20app/near/284025150

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

File: napari/_qt/dialogs/qt_plugin_dialog.py [hunk 1]
```diff
@@ -689,7 +689,12 @@ def tag_unavailable(self, project_info: PackageMetadata):
             widget = self.itemWidget(item)
             widget.show_warning(
                 trans._(
-                    "Plugin not yet available for installation within the bundle application"
+                    "This plugin is not available on conda-forge, so cannot be"
+                    " installed into this bundle. Ask the plugin developer to "
+                    "publish their plugin on conda-forge! Or if you use the .zip "
```
Comment (nclack) [inline napari/_qt/dialogs/qt_plugin_dialog.py]:
```suggestion
                    " installed into this bundle. If you use the .zip "
```

I don't really like the idea of asking people to pester plugin developers even when it's a good recommendation.

**Comment by jni** [comment]
Note that trans._() strings need to be on one line.

**Comment by Czaki** [comment]
> Note that trans._() strings need to be on one line.

So I should split this on multiple `trans._()` call or break the formating rules?

**Comment by Carreau** [comment]
> Note that trans._() strings need to be on one line.

would `\` escaping end of line work ?

**Comment by Carreau** [comment]
With #4588 multiline strings should be supported/detected.

**Comment by Carreau** [comment]
> So I should split this on multiple `trans._()` call or break the formatting rules?

I believe preferably break the formatting rule, as having a single. 
I'd love to get @goanpeca feedback here and on https://github.com/napari/napari/pull/4588, to see if we can find a solution to get multiple line strings in, as it is a regular issue.

**Comment by goanpeca** [comment]
>So I should split this on multiple trans._() call or break the formating rules?

Hi @Czaki, no that would not be advisable, as one needs to translate sentences, not chunks of sentences (that might loose the context) and language grammar and syntax might make words appear in a different chunk.

>I'd love to get @goanpeca feedback here and on https://github.com/napari/napari/pull/4588, to see if we can find a solution to get multiple line strings in, as it is a regular issue.

I need to take a look at that PR @Carreau thanks for working on it! Using a single line, introduced that annoyance  for large strings but was the easiest solution at the time. If the string parser you are working on #4588 solves that, then awesome, we can reformat the codebase to conform to the given line limit

**Comment by goanpeca** [comment]
Hi @Czaki thanks for the PR and the feedback.

I think the target audience for the bundle does not really need to know (or care for that matter) about `pip only plugins` of `githug`or `tags`.

This message feels like it is intended at a different audience so I am hesitant to add this.

I would rather prefer we had some sort of submission mechanism (not using Github) so users could "vote" or at least show the interest in any particular plugin.

**Comment by Czaki** [comment]
> I think the target audience for the bundle does not really need to know (or care for that matter) about `pip only plugins` of `githug`or `tags`.

This PR comes from plugin creator question. There is a need for clear feedback for plugin creator why he cannot install his plugin in bundle.

**Comment by goanpeca** [comment]
>This PR comes from plugin creator question. There is a need for clear feedback for plugin creator why he cannot install his plugin in bundle.

That is still not the intended target audience, and the mechanism to convey this information should then be different, be it documentation or a more continuous and direct contact with plugin developers.

**Comment by sofroniewn** [comment]
I don't love the specificity of pointing people the the 0.4.16 tag, or talking about briefcase and zips. Maybe instead we can just have a link to a documentation page

```
"Plugin not yet available for installation within the bundle application. For more information see napari.org/????"
```

That might allow folks that want to learn more a place to find more, without providing too much detail that will only confuse most people.

**Comment by sofroniewn** [comment]
We also might want to wait on this PR until after the packaging NAP has been resolved, and then we can revisit

**Comment by psobolewskiPhD** [comment]
In light of the plugin manager redesign 
https://github.com/napari/napari/pull/5198
split off of plugin manager:
https://github.com/napari/napari/pull/5743
and the bundle changes
https://github.com/napari/napari/pull/5804

Is this still `a thing` @Czaki ?
❤️

**Comment by Czaki** [comment]
At least not in this form.

**Pull request closed without merging**